app/controllers/spree/reviews_feeds_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Spree
+  class ReviewsFeedsController < Spree::StoreController
+    def show
```

Comment:
shouldn't this be an index?

Reply:
@kennyadsl I don't think so, the "reviews feed" is a singleton (i.e. there's only one reviews feed in the entire app, it's not a collection), so when you call `/reviews_feed.xml` you are effectively "showing" the reviews feed.

Reply:
This logic makes sense, I think we are just getting a little bit off the default Rails RESTful actions, where `show` needs the id of the resource you want to display. For what is worth (I'm also fine with this solution), I think the best thing here is to have an `index` action with a controller like `Spree::GoogleFeed::ReviewsController`. This way, it will also be easier to extend it with other feeds if needed later. Again, I'm fine with this, so going to approve.